src/index.spec.js

```diff
@@ -50,6 +50,33 @@ describe('provideState', () => {
     })
   })
 
+  describe('resetState', () => {
+    it('is called to reset the state to its initial values', () => {
+      const props = { bar: 'baz' }
+      const { effects, getInjectedState } = makeTestInstance(
+        {
+          initialState: () => ({
+            foo: 'bar',
+          }),
+          effects: {
+            changeState (_, value) {
+              return { foo: value }
+            },
+            reset () {
+              this.resetState()
```

Comment:
I think it would be nice for this helper to be available outside effects, so it could be used directly without having to create a proxy effect:

```js
<form onReset={resetState}>
```

Reply:
> I think it would be nice for this helper to be available outside effects, so it could be used directly without having to create a proxy effect:
> 
> ```js
> <form onReset={resetState}>
> ```

done